django/test/testcases.py

```diff
@@ -1345,3 +1351,26 @@ def setUpClass(cls):
     def tearDownClass(cls):
         super().tearDownClass()
         cls._lockfile.close()
+
+
+class WidgetTestCase(SimpleTestCase):
+    """
+    For testing widget rendering in Django's test suite (not a public API).
+    """
```

Comment:
Might be worth moving to a `/tests/utils.py` submodule instead of defining it here since it's not meant to be a public API.